src/components/Circle/index.js

```diff
@@ -60,16 +64,8 @@ const Circle = props => {
       return null;
     }
 
-    let blockySize = 25;
-    let blockyScale = 4;
-    if (size === 'small') {
-      blockySize = 13;
-      blockyScale = 4;
-    }
-    if (size === 'large') {
-      blockySize = 40;
-      blockyScale = 5;
-    }
+    let blockySize = 15;
```

Comment:
Awesome - keeping the blockyscale fixed is better as now we actually have a consistent blockie.